ELK_Listener/ci_listener.yaml

```diff
@@ -0,0 +1,352 @@
+- job-template:
+    name: 'CI-Metrics-Data-Listener'
+    description: |
+        Managed by Jenkins Job Builder. Do not edit via web
+    concurrent: false
+    node: master
+    triggers:
+        - ci-trigger:
+            jms-selector: "(CI_TYPE = '{index}' AND method = 'build' AND scratch = FALSE) OR (CI_TYPE = 'ci-metricsdata')"
+    builders:
+        - shell: |
+            #!/bin/bash
+            kinit -k -t $KEYTAB $PRINCIPAL
```

Comment:
Excellent.  The kinit allows the script to query the PDC without being asked to log in.